apps/backoffice-v2/src/pages/MerchantMonitoringUploadMultiple/hooks/useMerchantMonitoringUploadMultiplePageLogic/batch-report-template.csv

```diff
@@ -0,0 +1,3 @@
+websiteUrl,countryCode,lineOfBusiness,parentCompanyName,merchantName,correlationId
```

Comment:
Shouldn't reflect whats optional and what's not? it can be in help tooltip maybe

Reply:
Wasn't part of the design but you can ask Nitzan and we'll add it in the future